DuckDuckGo/Menus/MainMenuActions.swift

```diff
@@ -797,6 +797,10 @@ extension MainViewController {
     @objc func resetBookmarks(_ sender: Any?) {
         LocalBookmarkManager.shared.resetBookmarks()
         UserDefaults.standard.set(false, forKey: UserDefaultsWrapper<Bool>.Key.homePageContinueSetUpImport.rawValue)
+
+
+        UserDefaults.standard.set(false, forKey: UserDefaultsWrapper<Bool>.Key.bookmarksBarPromptShown.rawValue)
```

Comment:
ℹ️ I had to to add this to be consistent between new tests. When we reset bookmarks, I want to reset the show bookmarks bar prompt dialog, so it’s shown.

Reply:
That makes perfect sense 👍  but why are you adjusting onboardingFinished in the line below? It feels to me like a side effect (one wouldn't normally expect onboarding flag to be adjusted when you reset bookmarks, and especially not to have that flag set to `true`).

Reply:
I thought it was needed for the show bookmarks bar dialog to appear, but that is not the case. I removed it